### DISCUSSION TITLE
Get discussions option in a repository

### DISCUSSION THREAD

**Discussion by Daniellop1** [community/community#2839] [Discussions] [answered]
Hello, I would like to get the discussions option in this repository (http://github.com/dglobalcode/dglobal/). Thanks a lot.

**Comment by becca** [comment, marked as answer]
It is now possible for you to enable yourself on public repositories!
https://github.blog/changelog/2020-12-08-github-discussions-now-available-as-a-public-beta/

**Comment by Daniellop1** [comment]
I already know it. I just watched the event. Thank you very much for answering anyway
________________________________
